app/controllers/course/assessment/submission/answer/text_response/text_response_controller.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+class Course::Assessment::Submission::Answer::TextResponse::TextResponseController < \
+  Course::Assessment::Submission::Answer::TextResponse::Controller
+  load_resource :actable, class: Course::Assessment::Answer::TextResponse.name,
+                          singleton: true, through: :answer
+  before_action :set_text_response_answer
+
+  helper Course::Assessment::Submission::SubmissionsHelper.name.sub(/Helper$/, '')
+
+  def upload_files
+    success = @text_response_answer.class.transaction do
+      raise ActiveRecord::Rollback unless update_answer_files(upload_files_params)
+
+      true
+    end
+
+    render_response(success)
+  end
+
+  def delete_file
+    attachment_reference = @text_response_answer.attachments.find(delete_file_params[:attachment_id])
+    answer = @text_response_answer.acting_as
+
+    success = @text_response_answer.class.transaction do
+      answer.update!(last_session_id: session.id, client_version: delete_file_params[:client_version])
+      raise ActiveRecord::Rollback unless attachment_reference.destroy
```

Comment:
Why are we still using this kind of syntax? I just commented the exact same mistake you encountered in the previous line.